### PR TITLE
Jeffrey 2739 streamline benchmark creation

### PR DESCRIPTION
**Pull request by Jeffreyher**
https://jira.cms.gov/browse/QPPA-2739

Update the build:benchmarks script to skip the staging step and directly generate an updated benchmarks.json file. 
CMS sends complete copies of included benchmarks in the csv now; the merging step of previous JSON may be unnecessary.

changelog:
   - Update instructions on CONTRIBUTING.md for adding benchmarks. 
   - Update benchmarks/2018.json and 2019.json. Add ordering by measureId and submissionMethod
      - Will reduce git diff in the future, make it easier to see changes.
   - Update build-benchmarks script:
      - Update comments on what script does. No longer have to run parse-benchmarks-data then build-benchmarks. Just run build-benchmarks now.
      - Change script to accept performance year arg.
   - Update merge-benchmark-data-helpers:
      - remove overwrites of conflicting benchmarks w/ same composite key.
      - throws errors on conflicts.
   - Update merge-benchmark-data:
      - refactor so that __dirname dependency on depends on merge-benchmark-data's directory
         - makes spec files easier to reference correct fixtures.
   - Update parse-benchmarks-data:
      - add benchmark ordering
   - Delete unnecessary 2018 benchmark csvs.
   - Remove redundant 2018 benchmarks from layered JSON.
   - Update merge-benchmark-data-spec.

Test plan: 
- [x] verified working locally on JSON.
- [x] unit tested scripts

### CODE REVIEW

**Comment by Jeffreyher** [comment]
@kalvinwang I changed the merge to raise an error w/ merge conflicts for the benchmark unique constraints. Testing this on 2018, I got 77 merge conflicts.

If we're receiving complete copies of benchmarks from CMS now, could we just deprecate 2017-2018 updates, and only accept 2019 updates?

**Comment by Jeffreyher** [comment]
Force push to reorder git history. Introduce 2019.json reordering change first.

File: scripts/benchmarks/parse-benchmarks-data.js [hunk 1]
```diff
@@ -58,7 +59,8 @@ if (benchmarkYear && performanceYear) {
           if (benchmark) benchmarks.push(benchmark);
         });
 
-        process.stdout.write(JSON.stringify(benchmarks, null, 2));
+        const orderedBenchmarks = _.sortBy(benchmarks, ['measureId', 'submissionMethod']);
```
Comment (sheldon-b) [inline scripts/benchmarks/parse-benchmarks-data.js]: nice 👍

File: scripts/benchmarks/helpers/merge-benchmark-data-helpers.js [hunk 2]
```diff
@@ -31,20 +31,37 @@ const getBenchmarkKey = (benchmark) => {
...
+        });
+      } else {
+        mergedBenchmarks.set(benchmarkKey, benchmark);
+      }
     });
   });
 
-  return [...mergedBenchmarks.values()];
+  if (mergeConflicts.length > 0) {
+    throw new Error('Merge Conflicts: ' + JSON.stringify(mergeConflicts[0]));
```
[the remark this answers is not in the document]
Comment (Jeffreyher) [reply scripts/benchmarks/helpers/merge-benchmark-data-helpers.js]: Ooo, this is a typo. Meant to return all the errors.

**Pull request merged**